docs/compass/03-01-security.md

```diff
@@ -217,3 +218,32 @@ foo@bar.com:
 **Scopes**
 
 Scopes are added to the authentication session in Tenant Mapping Handler. The handler gets not only `tenant`, but also `scopes`, which are fixed regarding of type of the object (Application / Runtime). Application and Runtime are always have the same scopes defined.
+
+### One-time token
+
+**Used by:** Runtime/Application
+
+**Connector flow:**
+
```

Comment:
These are strange, I understand you wanted to avoid H4 but maybe that is another reason why that doc could be split into a few ones. I would also rather use full sentences than such "headings" "**Used by:** Runtime/Application".

Reply:
Good point, again, I'll leave it to @klaudiagrz.